cpp/tests/utilities/type_lists.hpp

```diff
@@ -87,6 +90,39 @@ auto make_type_param_vector(std::initializer_list<T> const& init_list)
   return vec;
 }
 
+template <typename TypeParam, typename T>
+typename std::enable_if<cudf::is_timestamp_t<TypeParam>::value, std::vector<TypeParam>>::type
+make_type_param_vector(std::initializer_list<T> const& init_list)
+{
+  std::vector<TypeParam> vec(init_list.size());
+  std::transform(std::cbegin(init_list), std::cend(init_list), std::begin(vec), [](auto const& e) {
+    return TypeParam{typename TypeParam::duration{e}};
+  });
+  return vec;
+}
+
+/**
+ * @brief Convert the numeric value of type T to a fixed width type of type TypeParam.
+ *
+ * @param init_value Value used to initialize the fixed width type
+ * @return A fixed width type - [u]int32/float/duration etc. of type TypeParam with the
+ *         value specified
+ */
```

Comment:
```suggestion
/**
 * @brief Convert the numeric value of type T to a fixed width type of type TypeParam.
 *
 * This function is necessary because some types (such as timestamp types) are not directly constructible 
 * from numeric types. This function is offered as a convenience to allow implicitly constructing 
 * such objects from numeric values. 
 *
 * @param init_value Value used to initialize the fixed width type
 * @return A fixed width type - [u]int32/float/duration etc. of type TypeParam with the
 *         value specified
 */
```

Reply:
@sriramch just make this doc change and then we can get this merged.

Reply:
done.